### ISSUE TITLE
In Touchable highlight page, no success info while invoking ‘A touchable highlight counter’ button.

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#230]
Found during accessibility grading.

Tracking internal bug: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/40128776
"Narrator should announce about the available number present inside the button."
- [x] "Note: Same issue is observed in ‘Touchable Opacity’."

**Comment by AgneLukoseviciute** [comment]
`accessibilityValue` should help
